rpc.go

```diff
@@ -252,6 +270,18 @@ func (t *Topic) watch() {
 	}
 }
 
+func (t *Topic) republishTo(peer.ID) {
+	t.lk.Lock()
+	for _, m := range t.ongoing {
+		go func(m ongoingMessage) {
```

Comment:
Log message below this line would be nice. Something like: `log.Debugf("%s republishing for %s", t.t, to)`, where `to` (or similar) is `peer.ID`.